x-pack/plugins/ml/server/saved_objects/service.ts

```diff
@@ -181,6 +208,7 @@ export function jobSavedObjectServiceFactory(
     return jobObject;
   }
 
+  // CHANGE!!!!!!!!!! this looks like a dupe of _getJobObjects
```

Comment:
Did you want to do this in this PR?

Reply:
on second review the two functions are not the same. will remove the comment